src/ProjectTeamCreator.sol

```diff
@@ -0,0 +1,114 @@
+
+    Project public table;
+
+    uint256 public projectTeamAdminHatId;
+
+    Whitelist internal whitelist;
+
+    bool public openAccess;
+
+    constructor(address _hats, address _hatsModuleFactory, address _hatsPassthrough, address _projectTeam, address _gnosisSingleton, address _gnosisSafeProxyFactory, address _table, address _whitelist) Ownable(msg.sender) {
+        hats = IHats(_hats);
+        projectTeam = ProjectTeam(_projectTeam);
+        gnosisSingleton = _gnosisSingleton;
+        hatsPassthrough = _hatsPassthrough;
+        gnosisSafeProxyFactory = GnosisSafeProxyFactory(_gnosisSafeProxyFactory);
+        hatsModuleFactory = HatsModuleFactory(_hatsModuleFactory);
+
+        table = Project(_table);
+        whitelist = Whitelist(_whitelist);
+    }
+
+    function setProjectTeamAdminHatId(uint256 _projectTeamAdminHatId) external onlyOwner() {
+        projectTeamAdminHatId = _projectTeamAdminHatId;
+    }
+
+    function setOpenAccess(bool _openAccess) external onlyOwner() {
+        openAccess = _openAccess;
+    }
+
+    function createProjectTeam(string memory adminHatURI, string memory managerHatURI, string memory memberHatURI, string calldata title, uint256 quarter, uint256 year, uint256 MDP, string calldata proposalIPFS, address lead, address[] memory members) external onlyOwner() payable returns (uint256 tokenId, uint256 childHatId) {
+        require(whitelist.isWhitelisted(lead) || openAccess, "Only whitelisted addresses can create a Project Team");
+
+
+        bytes memory safeCallData = constructSafeCallData(lead);
+        GnosisSafeProxy gnosisSafe = gnosisSafeProxyFactory.createProxy(gnosisSingleton, safeCallData);
+
+        //admin hat
+        uint256 teamAdminHat = hats.createHat(projectTeamAdminHatId, adminHatURI, 1, address(gnosisSafe), address(gnosisSafe), true, "");
+        hats.mintHat(teamAdminHat, address(this));
+
+        //manager hat
+        uint256 teamManagerHat = hats.createHat(teamAdminHat, managerHatURI, 8, address(gnosisSafe), address(gnosisSafe), true, "");
+
+        hats.mintHat(teamManagerHat, lead);
+        // loop through members and mint hats, before the safe has control
+        hats.transferHat(teamAdminHat, address(this), address(gnosisSafe));
+
+        //member hat
+        uint256 teamMemberHat = hats.createHat(teamManagerHat, memberHatURI, 1000, address(gnosisSafe), address(gnosisSafe), true, '');
+        for (uint i = 0; i < members.length; i++) {
+            hats.mintHat(teamMemberHat, members[i]);
+        }
+
+        //member hat passthrough module (allow admin hat to control member hat)
+        PassthroughModule memberPassthroughModule = PassthroughModule(deployModuleInstance(hatsModuleFactory, hatsPassthrough, teamMemberHat, abi.encodePacked(teamManagerHat), "", 0));
+
+
+        hats.changeHatEligibility(teamMemberHat, address(memberPassthroughModule));
+        hats.changeHatToggle(teamMemberHat, address(memberPassthroughModule));
+
+        //payment splitter
```

Comment:
We can probably remove the payment splitter. Any reason why the project lead gets 1%?